tests/run-pass/unsupported_foreign_function.rs

```diff
@@ -0,0 +1,11 @@
+// compile-flags: -Zmiri-panic-on-unsupported
+
+fn main() {
+    extern "Rust" {
+        fn foo();
+    }
+
+    unsafe {
+        foo();
+    }
+}
```

Comment:
Maybe move this test to the "panic" folder, or otherwise add "panic" to its name.